Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.py

```diff
@@ -32,6 +32,12 @@ def __init__(self, credentials: dict, use_oauth: bool = False, client_id: str =
             self.password = credentials.get('password')
             self.auth = (self.username, self.password)
 
+        if '@' in client_id:  # for use in OAuth test-playbook
+            self.client_id, refresh_token = client_id.split('@')
+            integration_context = get_integration_context()
```

Comment:
is there a reason for that? will there be anything in the context?

Reply:
Changed that. thanks!